server/scripts/get_players.py

```
@@ -2,17 +2,68 @@
+        143, 134, 135, 136, 137, 138, 139, 140, 141, 120
+    ]
+    
+    player_positions = {}
+    print(f"Fetching active player positions from MLB StatsAPI for {year}...")
+
+    infield_positions = {"C", "1B", "2B", "3B", "SS", "DH", "TWP"}
+    outfield_positions = {"LF", "CF", "RF"}
+
+    for tid in team_ids:
+        try:
+            resp = requests.get(
```

Ruff (0.14.3)

Probable use of `requests` call without timeout

(S113)